examples/impute/plot_iterative_imputer_variants_comparison.py

```diff
@@ -56,7 +56,7 @@
 from sklearn.pipeline import make_pipeline
 from sklearn.model_selection import cross_val_score
 
-N_SPLITS = 5
+N_SPLITS = 3
```

Comment:
a 3 fold doesn't really give us any reliable results, and that's why we've increased the default from 3 to 5 a few versions ago, I'd stick with 5 as a minimum, since people copy/paste code.